src/App.jsx

```diff
@@ -1,18 +1,22 @@
 import "./App.css";
 import { useState } from 'react';
-import TodoList from './todolist';
-import AddTodoForm from './addtodoform';
+import TodoList from './Todolist';
+import AddTodoForm from './AddTodoForm';
```

Comment:
make sure your imports are correctly referencing the file. In this situation its throwing an error because the "d" is lowercase so make sure you import "./AddToDoForm".